app/views/underpayments/UnderpaymentDetailsView.scala.html

```diff
@@ -25,13 +25,13 @@
 @(form: Form[_], underpaymentType: String, backLink: Call)(implicit request: Request[_], messages: Messages)
 
 @layout(
-    pageTitle = messages(s"underpaymentDetails.$underpaymentType.pageTitle"),
+    pageTitle = messages(s"underpaymentDetails.$underpaymentType.pageTitle") + " " + messages("underpaymentDetails.dutyMessageEnding"),
```

Comment:
You could just add the space to the dutyEnding message

Reply:
pretty sure we tried it and it gets cleaned as a spare whitespace